sdk/trace/trace_test.go

```diff
@@ -217,9 +220,16 @@ func TestSetSpanAttributesOverLimit(t *testing.T) {
 		},
 		ParentSpanID: sid,
 		Name:         "span0",
-		Attributes: map[string]interface{}{
-			"key1": core.Value{Type: core.STRING, String: "value3"},
-			"key4": core.Value{Type: core.STRING, String: "value4"}},
+		Attributes: []core.KeyValue{
+			{
+				Key:   core.Key{Name: "key1"},
+				Value: core.Value{Type: core.STRING, String: "value3"},
```

Comment:
(Sorry for the late review - I was just working on this same thing in our exporter)
It would be nice to see attribute types other than strings in the tests here.

Reply:
sure. I'll add that with another PR